webapp/graphite/metrics/views.py

```diff
@@ -100,8 +104,29 @@ def find_view(request):
   format = queryParams.get('format', 'treejson')
   local_only = int( queryParams.get('local', 0) )
   wildcards = int( queryParams.get('wildcards', 0) )
-  fromTime = int( queryParams.get('from', -1) )
-  untilTime = int( queryParams.get('until', -1) )
+
+  tzinfo = pytz.timezone(settings.TIME_ZONE)
+  if 'tz' in queryParams:
+    try:
+      tzinfo = pytz.timezone(queryParams['tz'])
+    except pytz.UnknownTimeZoneError:
+      pass
+
+  if 'now' in queryParams:
+    now = parseATTime(queryParams['now'], tzinfo)
+  else:
+    now = datetime.now(tzinfo)
+
+  if 'from' in queryParams and str(queryParams['from']) != '-1':
+    fromTime = int(epoch(parseATTime(queryParams['from'], tzinfo, now)))
```

Comment:
How will it work if `from` is timestamp? Maybe we should make some shortcut for this ?